src/lib/containers/query_wrapper_plot_nav/QueryWrapperPlotNav.tsx

```diff
@@ -28,6 +28,12 @@ type OwnProps = {
   sql: string
   limit?: number
   shouldDeepLink?: boolean
+  /** If onQueryChange is set, the callback will be invoked when the Query changes */
+  onQueryChange?: (newQueryJson: string) => void
+  /** If onQueryResultBundleChange is set, the callback will be invoked when the QueryResultBundle changes */
+  onQueryResultBundleChange?: (newQueryResultBundleJson: string) => void
+  /** If initQueryJson is set, it will be the Query used in the initial QueryBundleRequest */
+  initQueryJson?: string
```

Comment:
Provide way to initialize the Query (set by SWC based to either the default Query, or it uses the encoded Query from the URL)

Reply:
I think I understand, this is a string because otherwise you would need a JSInterop compatible Query object, and at that point we're replicating models that we already get from the backend.